src/main/java/core/basesyntax/db/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.db.service.impl;
+
+import core.basesyntax.db.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> readFromFile(String path) {
+        List<String> lines;
+        try {
+            lines = Files.readAllLines(Path.of(path));
+            lines.remove(0);
```

Comment:
I know why you remove first line here. But if i wouldn't know the requirements, i would think that this is a hardcore or a mistake. Let's add to the name of this class "Csv" or move this removing to another class